# disallow block-scoped variable declarations (es/no-block-scoped-variables)

This rule reports ES2015 block-scoped variable declarations as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

<eslint-playground type="bad" code="/*eslint es/no-block-scoped-variables: error */
let a = 1
const b = 2
" />

## ðŸ“š References

- [Rule source](https://github.com/mysticatea/eslint-plugin-es/blob/v3.0.1/lib/rules/no-block-scoped-variables.js)
- [Test source](https://github.com/mysticatea/eslint-plugin-es/blob/v3.0.1/tests/lib/rules/no-block-scoped-variables.js)
