# disallow property shorthands (es/no-property-shorthands)

- ðŸ”§ The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.

This rule reports ES2015 property shorthands as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

<eslint-playground type="bad" code="/*eslint es/no-property-shorthands: error */
let obj = {
    a,
    b() {}
}
" />

## ðŸ“š References

- [Rule source](https://github.com/mysticatea/eslint-plugin-es/blob/v3.0.1/lib/rules/no-property-shorthands.js)
- [Test source](https://github.com/mysticatea/eslint-plugin-es/blob/v3.0.1/tests/lib/rules/no-property-shorthands.js)
