# disallow RegExp `y` flag (es/no-regexp-y-flag)

This rule reports ES2015 RegExp `y` flag as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

<eslint-playground type="bad" code="/*eslint es/no-regexp-y-flag: error */
const r1 = /foo/y
" />

## ðŸ“š References

- [Rule source](https://github.com/mysticatea/eslint-plugin-es/blob/v3.0.1/lib/rules/no-regexp-y-flag.js)
- [Test source](https://github.com/mysticatea/eslint-plugin-es/blob/v3.0.1/tests/lib/rules/no-regexp-y-flag.js)
