# disallow spread elements (es/no-spread-elements)

This rule reports ES2015 spread elements as errors.

## Examples

â›” Examples of **incorrect** code for this rule:

<eslint-playground type="bad" code="/*eslint es/no-spread-elements: error */
const a1 = [1, 2, ...array]
foo(...a, ...b)
" />

## ðŸ“š References

- [Rule source](https://github.com/mysticatea/eslint-plugin-es/blob/v3.0.1/lib/rules/no-spread-elements.js)
- [Test source](https://github.com/mysticatea/eslint-plugin-es/blob/v3.0.1/tests/lib/rules/no-spread-elements.js)
